### ISSUE TITLE
Is there a way to download ImageNet weights for WRN-28-2?

### ISSUE THREAD

**Issue by varunnair18** [brain-research/realistic-ssl-evaluation#28]
I see that the repository provides a method to download ImageNet in a 32x32 format, however, would you have access to the weights for the WRN-28-2 model that was trained on ImageNet and used in the transfer learning section of the paper?

I would like to use the ImageNet weights in another application, but if you had a link or another method for me to download them it would save much time and cost in training the network on ImageNet from scratch.

Thanks!

**Comment by craffel** [comment]
No. Note that since it is Imagenet 32x32, pre-training on imagenet is not really any more expensive than any of the other semi-supervied training runs.

**Comment by varunnair18** [comment]
@craffel Would you know approximately how long (using a single GPU) I should expect the model to train for on the ImageNet 32x32 to achieve the same model weights that you use for the transfer learning experiments?

**Comment by craffel** [comment]
@DoctorTeeth might know.

**Comment by TheRevanchist** [comment]
It took me around half a day to reproduce their ImageNet results (using their code) in a Titan Xp.

**Comment by DoctorTeeth** [comment]
Oh gosh, it was a long time ago.
I want to say that around a day sounds right.
I don’t remember having to wait longer than that for the imagenet model to
finish training.

**Comment by craffel** [comment]
Yeah a day-ish sounds right. This would have been on a k80.

**Comment by varunnair18** [comment]
Thanks for that, also while trying to use the training command in `/runs/section-4-3-imagenet-pre-training.yml`, I found that I didn't have the zca_mean.npy file necessary to perform the ZCA whitening. Is there a command that I need to run to generate it or somewhere online I can download it?

I was able to successfully follow the instruction for creating tfrecords for ImageNet, but didn't get this part.
